docs/configuration.md

```diff
@@ -611,6 +611,13 @@ You can optionally also automatically redirect users to the SSO login with [PAPE
 : Only applies to regular (non-SSO) accounts. See the corresponding
 [django-allauth documentation](https://docs.allauth.org/en/latest/account/configuration.html)
 
+#### [`PAPERLESS_SESSION_COOKIE_AGE=<bool>`](#PAPERLESS_SESSION_COOKIE_AGE) {#PAPERLESS_SESSION_COOKIE_AGE}
```

Comment:
This says bool instead of int.  Confused me for a second

Reply:
Oh thanks I'll fix